### PR TITLE
Add tool to sync string changes in codebase with server strings.

### PR DESCRIPTION
**Pull request by tnagorra**
- [x] Need to update the commit names
- [x] Need to rebase to develop

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by frozenhelium — APPROVED** [review]

File: app/scripts/lint-translations.js
Comment (frozenhelium) [inline app/scripts/lint-translations.js:1]: This needs to be removed

File: app/package.json [hunk 1]
```diff
@@ -22,7 +23,7 @@
         "lint:js": "eslint src",
         "lint:css": "stylelint \"./src/**/*.css\"",
         "lint:unused": "unimported",
-        "lint:translation": "node ./scripts/translator.js",
+        "lint:translation": "node ./scripts/lint-translations.js",
```
Comment (tnagorra) [inline app/package.json]:
```suggestion
        "lint:translatte": "yarn translatte lint ./src/**/i18n.json ../packages/ui/src/**/i18n.json",
```

File: app/scripts/lint-translations.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { isDefined, listToMap, mapToList } from '@togglecorp/fujs';
```
Comment (tnagorra) [inline app/scripts/lint-translations.js]: Let's remove this file

File: app/scripts/translatte/commands/generateMigration.ts [hunk 3]
```diff
@@ -24,96 +29,85 @@ function generateMigration(
...
-        rightRemainder: currentState,
-    } = oneOneMapping(
-        prevState,
-        currentState,
-        (item) => `${item.namespace}:${item.key}`,
-        (prev, current) => prev.value === current.value,
-    ));
+    console.info('prevState length', prevState.length);
+    console.info('currentState length', currentState.length);
+    console.info('Total change', Math.abs(prevState.length - currentState.length));
```
Comment (tnagorra) [inline app/scripts/translatte/commands/generateMigration.ts:35]: Let's remove the console meant for debugging

File: app/scripts/translatte/commands/generateMigration.ts [hunk 4]
```diff
@@ -172,7 +168,7 @@ async function generate(
     const lastMigration = migrationFilesAttrs[migrationFilesAttrs.length - 1];
 
     const migrationContent: MigrationFileContent = {
-        parent: lastMigration.migrationName,
+        parent: lastMigration?.migrationName,
```
Comment (tnagorra) [inline app/scripts/translatte/commands/generateMigration.ts:173]: This should always exist as we have already check for the length

Comment (frozenhelium) [reply app/scripts/translatte/commands/generateMigration.ts:173]: No No! This is correct. I get error otherwise.

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**